Tita-Clone/Tita-Clone/Resources/SceneDelegate.swift

```diff
@@ -19,7 +19,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             
         let navigationController = UINavigationController(rootViewController: mainViewController)
         navigationController.navigationBar.isHidden = true
-        window?.rootViewController = navigationController
+        window?.rootViewController = TabBarViewController()
```

Comment:
```suggestion
        window?.rootViewController = LoginViewController()
```
로그인 버튼 클릭 시 `TabBarViewController`로 넘어갈 수 있게 하는 게 좋겠네요~